inference-engine/include/gpu/gpu_ocl_wrapper.hpp

```diff
@@ -39,7 +39,7 @@
 # pragma GCC system_header
 #endif
 
-#include <CL/cl2.hpp>
```

Comment:
I'm not sure that this is a good idea. We don't include ocl headers into distribution, so we rely on user's OCL version. And I believe in ubuntu 18 apt package doesn't have opencl.hpp

Reply:
good note, thanks, fixed